src/main/java/com/codecool/barbershop/barbershop/client/Client.java

```diff
@@ -2,35 +2,45 @@
 
 import javax.validation.constraints.Email;
 import javax.validation.constraints.NotNull;
+
 import lombok.*;
 
 import javax.persistence.*;
+import java.util.Date;
 
 @AllArgsConstructor
 @NoArgsConstructor
 @Setter
 @Getter
 @Entity
 @ToString
-@Table(name="clients")
+@Table(name = "clients")
 public class Client {
 
     @Id
     @GeneratedValue(strategy = GenerationType.IDENTITY)
     private long clientId;
 
-    @NotNull private String firstName;
-
-    @NotNull private String lastName;
+    @NotNull
+    private String firstName;
 
-    @NotNull @Email
-    @Column(unique = true) private String email;
+    @NotNull
+    private String lastName;
 
-    @NotNull private String phoneNo;
+    @NotNull
+    @Email
+    @Column(unique = true)
+    private String email;
 
-//    TODO create date & update date
+    @NotNull
+    private String phoneNo;
 
+    @Temporal(TemporalType.TIMESTAMP)
```

Comment:
Does that mean that the Date type will get converted into the Timestamp in the database?